### PR TITLE
MEA-1843 Add anchor target prop

### PR DESCRIPTION
**Pull request by okanoktartw**
Customers in Turkey are being redirected to [transferwiseturkiye.com.tr](transferwiseturkiye.com.tr) which renders [transferwise.com](transferwise.com) in an iframe. We finally convinced partner to get users out of the iframe once they go to login-flow. So when they click login or register or are already logged in, they will be redirected to transferwise.com/login or transferwise.com/register and will continue from there. 

To minimise the change, we have decided to set `target="_top"` for login and register links in homepage. This will get users out of the iframe and redirect them to transferwise.com.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transferwise/public-navigation",
-  "version": "4.0.2",
+  "version": "4.0.3",
```
Comment (KarlEST) [inline package.json:3]: Probably should be minor version as we don't fix bugs and add functionality?

Comment (okanoktartw) [reply package.json]: Agree, updated as minor bump 👍

**Comment by okanoktartw** [comment]
> Same Q as on Homepage: Will it work with different domains?

I've added sandbox attribute `"allow-top-navigation"` in our turkey website [here](https://github.com/transferwise/tr-landing-page/blob/master/js/index.js#L15) and tested with major browsers successfully. According to this page, all browsers seem to be supporting sandbox attributes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# v4.0.3
```
Comment (KarlEST) [inline CHANGELOG.md:1]: Need to update this one as well 😉

**Pull request merged**